index.js

```diff
@@ -16,6 +16,23 @@ module.exports.pitch = function (request) {
 	var options = loaderUtils.getOptions(this) || {};
 
 	validateOptions(require('./options.json'), options, 'Style Loader')
+	options.hmr = typeof options.hmr === 'undefined' ? true : options.hmr;
```

Comment:
Please use `short-circuit` here (`options.hmr = options.hmr || true`)

Reply:
and a space `\n`

Reply:
I don't understand how `options.hmr = options.hmr || true` is going to work. We'll never be able to set this flag to `false`, right?